modules/book.js

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable import/prefer-default-export */
+const form = document.querySelector('form');
+const bookList = document.getElementById('bookList');
+export class Abooks {
+  constructor(books) {
+    this.books = books;
+    this.local = () => {
+      if (localStorage.getItem('books')) {
+        books = JSON.parse(localStorage.getItem('books'));
+      }
+    };
+
+    this.displayBooks = () => {
+      if (localStorage.getItem('books')) {
+        books = JSON.parse(localStorage.getItem('books'));
+      }
+      bookList.className = 'bookList';
+      bookList.innerHTML = '';
+      books.forEach((book, index) => {
+        const li = document.createElement('li');
+        li.className = 'lineBook';
+        li.innerHTML = ` " ${book.title} " by ${book.author} <button class="removeBtn" data-index="${index}">Remove</button>`;
+        bookList.appendChild(li);
+      });
+    };
+
+    this.submitbtn = () => {
+      const title = document.getElementById('bookTitle').value;
+      const author = document.getElementById('authorName').value;
```

Comment:
- [ ] Something has broken the code, the user is no longer able to add and delete books. Kindly fix this